contracts/contracts/core/BidderRegistry.sol

```diff
@@ -51,6 +51,7 @@ contract BidderRegistry is
         feePercent = _feePercent;
         blockTrackerContract = IBlockTracker(_blockTracker);
         blocksPerWindow = _blocksPerWindow;
```

Comment:
Calling out issue we talked about over call w.r.t blocksPerWindow. This param should be hardcoded immutable in the contracts, or the contracts should be refactored such that we can change the param without breaking accounting logic